scripts/cibuild.rb

```diff
@@ -105,6 +118,33 @@ def verify_file (f)
       return [f, error]
     end
 
+    if github_link?(link) then
+
+      if link.index('https://github.com/issues?q=') then
```

Comment:
Not sure what other parameters, if any, are valid for `/issues`, but the `q=` here seems overly specific; anything at `/issues` can be ignored from GH repo link parsing.